src/main/main.ts

```diff
@@ -17,7 +19,29 @@ log.transports.console.useStyles = true;
 log.transports.file.fileName = 'nova-spektr.log';
 log.transports.file.format = '{y}/{m}/{d} {h}:{i}:{s}.{ms} [{env}#{version}]-{processType} [{level}] > {text}';
 log.transports.file.level = 'info';
-log.transports.file.maxSize = 1048576 * 5; //5mb
+log.transports.file.maxSize = 1048576 * 5; //5mb;
+log.transports.file.archiveLogFn = (oldLogFile: LogFile): void => {
+  const file = oldLogFile.toString();
+  const info = path.parse(file);
+
+  try {
+    const date = new Date();
+    let newFileName = path.join(
+      info.dir,
+      info.name + '.' + date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate() + info.ext,
+    );
+    if (fs.existsSync(newFileName)) {
```

Comment:
I would suggest to use:
const date = new Date().toISOString();
in order to avoid duplication problem

<img width="532" alt="Screenshot 2023-07-26 at 11 01 12" src="https://github.com/novasamatech/nova-spektr/assets/40560660/ac390e44-aea8-42dc-b814-7eb93d0853a0">

Reply:
Looks better 
<img width="1032" alt="image" src="https://github.com/novasamatech/nova-spektr/assets/4741733/16b893d6-ad13-460d-89ad-53b6615e31c8">